gtk/gtk.go

```diff
@@ -9065,6 +9058,25 @@ func (v *TreeModelFilter) SetVisibleColumn(column int) {
 	C.gtk_tree_model_filter_set_visible_column(v.native(), C.gint(column))
 }
 
+// ConvertChildPathToPath is a wrapper around gtk_tree_model_filter_convert_child_path_to_path().
+func (v *TreeModelFilter) ConvertChildPathToPath(childPath *TreePath) *TreePath {
+	path := C.gtk_tree_model_filter_convert_child_path_to_path(v.native(), childPath.native())
+	if path == nil {
+		return nil
+	}
+	p := &TreePath{path}
+	runtime.SetFinalizer(p, (*TreePath).free)
+	return p
+}
+
+// ConvertChildIterToIter is a wrapper around gtk_tree_model_filter_convert_child_iter_to_iter().
+func (v *TreeModelFilter) ConvertChildIterToIter(childIter *TreeIter) (*TreeIter, bool) {
+	var iter C.GtkTreeIter
+	ok := gobool(C.gtk_tree_model_filter_convert_child_iter_to_iter(v.native(), &iter, childIter.native()))
+	t := &TreeIter{iter}
+	return t, ok
+}
```

Comment:
At parts like this I'm not sure if we should follow GTK or not. Other code parts return an error instead of bool as second return value.

see binding for gtk_combo_box_get_active_iter in combo_box.go

@founderio : want to pitch in?

Reply:
i create the issue #513 for later answer. i think actual is better we merged this and fix the return value later.